couchbase-persistence/couchbase-persistence-scala-sbt/project/build.properties

```diff
@@ -1 +1 @@
-sbt.version=1.3.13
+sbt.version=1.5.6
```

Comment:
And today's bid is 1.5.7!

Reply:
We need a condition for exit from this loop 😂